projects/1.20-fabric/assets/worldtools/worldtools/lang/zh_cn.json

```
@@ -0,0 +1,139 @@
+{
+  "text.autoconfig.worldtools.category.Advanced": "高级",
+  "text.autoconfig.worldtools.category.Debug": "调试",
+  "text.autoconfig.worldtools.category.Entity": "实体",
+  "text.autoconfig.worldtools.category.General": "基本",
+  "text.autoconfig.worldtools.category.Render": "渲染",
+  "text.autoconfig.worldtools.category.World": "世界",
+  "text.autoconfig.worldtools.option.advanced": "高级",
+  "text.autoconfig.worldtools.option.advanced.anonymousMode": "隐藏游戏界面坐标",
+  "text.autoconfig.worldtools.option.advanced.anonymousMode.@Tooltip": "启用后，捕获的坐标信息将在游戏界面中隐藏。§l注意§r：这不会影响实际数据！",

```

```suggestion
  "text.autoconfig.worldtools.option.advanced.anonymousMode.@Tooltip": "启用后，捕获的坐标信息将在游戏界面中隐藏。注意：这不会影响实际数据！",
```
一般不需要更改原文字体格式
此类问题请全文通查并修改

---

原文使用全大写来强调文本，请问翻译之后出了这么加粗，怎么处理比较好

---

建议不动，因为中文里的任何强调都需要引入外部符号或格式调整
如果你坚持要保留强调的话，那么加粗也可